src/parsers/pairs.ts

```diff
@@ -104,8 +110,17 @@ export function getInterpolatorForPair({ from, to }: CSSPairs) {
         interpolator: interpolateTransforms,
         property: fromProperty,
         values: {
-          from: fromValue,
-          to: toValue,
+          from:
+            // Add translateZ(0) to transform if it's not already present
+            // and if hardware acceleration is enabled.
+            fromValue.indexOf('translateZ') === -1 &&
```

Comment:
Decided to use `indexOf` in lieu of `includes` for IE11 compatibility. Not sure if folks have opinions here (i.e. consumer should polyfill `String.prototype.includes`, we just shouldn't care about IE11, etc)?